cdn/package.json

```diff
@@ -6,12 +6,13 @@
   "main": "src/spa.js",
   "scripts": {
     "test": "echo \"Error: no test specified\" && exit 1",
-    "webpack": "npx webpack --progress --config ./webpack.config.js",
+    "webpack": "rm -rf ../build && npx webpack --progress --config ./webpack.config.js",
```

Comment:
[suggestion] This breaks local watch builds. Can we remove the `rm -rf ../build && `? The build directory shouldn't exist in the scope of a GH workflow, right?

Reply:
Certain actions do trigger multiple builds for one reason or another (like building with and without sourcemaps).  Could probably change but thats why that was added

Reply:
All good....we will just have to accept that the watch does not work right now.

Reply:
for now will just move it out of the base `webpack` script